keystorage/hashicorp/src/test/java/tech/pegasys/signers/hashicorp/MockedVertxHashicorpConnectionFactoryTest.java

```diff
@@ -24,84 +22,24 @@
 import java.io.IOException;
 import java.util.Optional;
 
-import io.vertx.core.Vertx;
-import io.vertx.core.http.HttpClientOptions;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
-import org.mockito.ArgumentCaptor;
 
 class MockedVertxHashicorpConnectionFactoryTest {
 
-  final Vertx mockedVertx = mock(Vertx.class);
-  final HashicorpConnectionFactory connectionFactory = new HashicorpConnectionFactory(mockedVertx);
-
-  final ArgumentCaptor<HttpClientOptions> clientOptionsArgCaptor =
-      ArgumentCaptor.forClass(HttpClientOptions.class);
+  final HashicorpConnectionFactory connectionFactory = new HashicorpConnectionFactory();
 
   final int CONFIGURED_PORT = 500;
   final String CONFIGURED_HOST = "Host";
 
   @Test
-  void serverAndPortFromConnectionConfigAreUsedToConstructHttpClient() {
```

Comment:
Why delete this test, it seems useful

Reply:
The vertx httpclient was used to initialise with host+port. The Java HttpClient is not initialized with host+port, rather host+port/path is specified when we perform `httpclient.send`. Hence this context of this test has changed.